contrib/devtools/update-cargo-lock.py

```
@@ -0,0 +1,54 @@
+
+    if had_config:
+        shutil.copy2(CARGO_CONFIG, CARGO_CONFIG_BAK)
+        os.remove(CARGO_CONFIG)
+        atexit.register(lambda: shutil.move(CARGO_CONFIG_BAK, CARGO_CONFIG) if os.path.exists(CARGO_CONFIG_BAK) else None)
+
+    if os.path.exists(CARGO_LOCK):
+        cargo_lock_bak = CARGO_LOCK + '.bak'
+        shutil.copy2(CARGO_LOCK, cargo_lock_bak)
+
+    try:
+        subprocess.check_call(['cargo', 'generate-lockfile'], cwd=ROOT_DIR)
```

Ruff (0.14.14)

[error] Starting a process with a partial executable path

(S607)